lib/index.js

```diff
@@ -47,7 +51,7 @@ function calculateHistogram(arr, numBins = 0, trimTailPercentage = 0.00) {
     dataCopy.forEach(item => {
         let binIndex = Math.floor((item - min) / binSize);
         // for values that lie exactly on last bin we need to subtract one
-        if (binIndex === numBins) {
+        if (binIndex === numBins || JSON.stringify(binIndex) === numBins) {
```

Comment:
Same comment as above. I don't think it is the module's responsibility to accept parameters of multiple types. The user can easily convert to correct type prior to calling the module